hypothesis-python/src/hypothesis/core.py

```diff
@@ -1054,6 +1055,11 @@ def find(
     # type: (...) -> Any
     """Returns the minimal example from the given strategy ``specifier`` that
     matches the predicate function ``condition``."""
+    note_deprecation(
+        "`find(s, f)` is deprecated in favor of `s.filter(f).example()`, which "
```

Comment:
I don't think this deprecation notice is really true. `find` is being deprecated in favour of not using this functionality because basically nobody does and we don't want to support it. `@given` has no real shrinking advantage over `find`, and I don't especially want to encourage people to use `.example()` _more_ than they already do.